submissions/saudatSalawudeen/html/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>Product Page</title>
+</head>
+<body>
+
+	<!-- HEADER -->
+	<header>
+		<div id="head">
+			<h1>ZARIS</h1>
+			<p><em>home of sexy bodice and underwears!</em></p>
+		</div>
+		<nav>
+			<p>
+				<a href="#">Home</a> |
+				<a href="#">About-Us</a> |
+				<a href="#">Contact-Us</a>
+			</p>
+			<form>
+				<p>
+					<label for=""></label>
+					<input type="search" name="" id="" placeholder="product search">
+				</p>
+			</form>
+		</nav>
+	</header>
+	<hr/>
+
+	<!-- PRODUCT SHOWCASE -->
+	<main>
+		<h2>LACE PANTS</h2>
+		<div class="container">
+			<div>
+				<img src="https://res.cloudinary.com/omobola/image/upload/v1618468059/dufuna/pd1.jpg"
```

Comment:
Instead of using a URL for the image src, kindly download this image and place it inside an `images` folder in your `html` folder. Then target the path to that image here

Reply:
@adaugochi , please i have done that and pushed another commit.